src/lib/stitching/gravity/stitching.ts

```diff
@@ -44,6 +48,32 @@ export const gravityStitchingEnvironment = (
           },
         },
       },
+      ArtistSeries: {
+        artists: {
+          fragment: gql`
+          ... on ArtistSeries {
+            artistIDs
+          }
+        `,
+          resolve: ({ artistIDs: ids }, args, context, info) => {
+            if (ids.length === 0) {
```

Comment:
If this condition is true, you might just return early with an empty response. Something like `if (ids.length === 0) return []` (the `[]` to match the array type of this field).

Reply:
Ah, you know we cargo-culted this from L85-L95 below, but in doing so we didn't retain the comment that gives the context for this decision.

Now that I'm reading that comment more closely, I'm not quite sure why this is an issue, like why wouldn't we just early-return? 🤔

Reply:
@anandaroop Do you mean just a plain `return` instead of `return []`?

Reply:
I was thinking `return []` as Matt was suggesting

Reply:
Early return added in 049dcb5679280a2f28d6509f90996691d5f1d343

After that I think this is good to merge?